submission/persistence-filesystem/src/test/kotlin/ac/uk/ebi/biostd/persistence/filesystem/fire/FireFilesServiceTest.kt

```diff
@@ -50,23 +50,26 @@ internal class FireFilesServiceTest(
             fireClient.findByMd5("the-md5")
         }
         verify(exactly = 0) {
-            fireClient.downloadByFireId("the-fire-id", "file.txt")
-            fireClient.setPath("the-fire-id", "/S-BSST/001/S-BSST1/Files/folder/file.txt")
-            fireClient.save(any(), "the-md5", 123L)
+            fireClient.downloadByFireId(any(), any())
+            fireClient.setPath(any(), any())
+            fireClient.save(any(), any(), any())
         }
     }
 
     @Test
-    fun `persist fire file found by md5 but no path`() {
-        val file = fireFile()
-        val fireApiFile = fireApiFile()
+    fun `persist fire file found by md5 but no path is set`() {
+        val file = fireFile(null)
```

Comment:
Perhaps using the named parameter is a good idea to know what the null is referring to in a single glance?